src/Lykke.Service.FeeCalculator.AzureRepositories/CashoutFee/CashoutFeeEntity.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using Lykke.AzureStorage.Tables;
+using Lykke.AzureStorage.Tables.Entity.Annotation;
+using Lykke.AzureStorage.Tables.Entity.ValueTypesMerging;
+using Lykke.Service.FeeCalculator.Core.Domain.CashoutFee;
+using Lykke.Service.FeeCalculator.Core.Domain.Fees;
+
+namespace Lykke.Service.FeeCalculator.AzureRepositories.CashoutFee
+{
+    [ValueTypeMergingStrategy(ValueTypeMergingStrategy.UpdateAlways)]
+    public class CashoutFeeEntity : AzureTableEntity, ICashoutFee
+    {
+        public string Id { get; set; }
+        public string AssetId { get; set; }
+        public double Size { get; set; }
+        public FeeType Type { get; set; }
+
+        internal static string GeneratePartitionKey() => "CashoutFee";
+        internal static string GenerateRowKey(string id) => id;
+
+
+        internal static CashoutFeeEntity Create(ICashoutFee fee)
+        {
+            string id = fee.Id ?? Guid.NewGuid().ToString();
```

Comment:
Is it correct generating a new ID here? Later it will imposible to find the record by this id without loading all records.

Reply:
its ok here